aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cu

```diff
@@ -295,6 +295,70 @@ std::tuple<Tensor, Tensor, Tensor> _svd_helper_cuda_lib(const Tensor& self, bool
   return std::make_tuple(U_working_copy, S_working_copy, VT_working_copy);
 }
 
+/*
+  The orgqr function allows reconstruction of an orthogonal (or unitary) matrix Q,
```

Comment:
Would a single orgqr note that could be referenced in multiple places be easier to maintain?

Reply:
We don't have proper C++ documentation for the internal functions. So these notes are useful for someone who skims through the source code and I think it's more beneficial to have it right there next to the function.